internal/provisioner/fake_client.go

```diff
@@ -210,9 +210,10 @@ func (c *FakeClient) RuntimeStatus(accountID, runtimeID string) (schema.RuntimeS
 			return schema.RuntimeStatus{
 				RuntimeConfiguration: &schema.RuntimeConfig{
 					ClusterConfig: &schema.GardenerConfig{
-						Name:   ptr.String("fake-name"),
-						Region: ptr.String("fake-region"),
-						Seed:   ptr.String("fake-seed"),
+						Name:     ptr.String("fake-name"),
+						Region:   ptr.String("fake-region"),
+						Seed:     ptr.String("fake-seed"),
+						Provider: ptr.String("aws"),
```

Comment:
The choice of `aws` instead of fake-provider is due to the way the `GardenerConfig` struct is unmarshalled. If an existing provider is not provided, an error is returned.